newrelic_beego.go

```diff
@@ -67,6 +67,13 @@ func NameTransaction(ctx *context.Context) {
 	} else {
 		path = reNumberIDInPath.ReplaceAllString(ctx.Request.URL.Path, ":id")
 	}
+
+	displayExplicitEnv := beego.AppConfig.DefaultString("newrelic_display_explicit_env", "")
```

Comment:
What is expected behaviour if parameter `newrelic_display_explicit_env` is set to string "false" ? I believe it should live with ":env" in the URIs. But in current implementation it will substitute it with "IL".
I suggest to have default value as "FALSE" and check if `strings.ToUpper(displayExplicitEnv ) == "TRUE"` (if yes then perform replacement).